homeassistant/components/tfl/__init__.py

```diff
@@ -0,0 +1,74 @@
+"""The Transport for London integration."""
+
+from __future__ import annotations
+
+import logging
+from urllib.error import HTTPError, URLError
+
+from tflwrapper import stopPoint
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryAuthFailed, ConfigEntryNotReady
+
+from .const import CONF_API_APP_KEY, DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+PLATFORMS: list[Platform] = [Platform.SENSOR]
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up Transport for London from a config entry."""
+
+    hass.data.setdefault(DOMAIN, {})
```

Comment:
Instead of this, please use this pattern https://developers.home-assistant.io/blog/2024/04/30/store-runtime-data-inside-config-entry

Reply:
updated